TFT/src/User/API/interfaceCmd.c

```diff
@@ -247,26 +233,21 @@ void sendQueueCmd(void)
         case 1:
           if (isPrinting() && infoMachineSettings.firmwareType != FW_REPRAPFW)  // Abort printing by "M0" in RepRapFirmware
           {
-            // firstly dequeue the gcode to avoid a possible reprocessing or infinite nested loop in case
-            // the function loopProcess() is invoked by the following function printPause()
-            char gcode[CMD_MAX_CHAR];
-            dequeueLastCmd(gcode);
-
-            if (printPause(true, true))  // if print successfully paused, send the gcode
-              sendDequeuedCmd(gcode, avoid_terminal);
-            return;
+            // pasue if printing form TFT and purge M0/M1 command.
+            if (infoFile.source < BOARD_SD )
+            {
+              purgeLastCmd(avoid_terminal);
+              if (!isPaused()) printPause(true, PAUSE_M0);
```

Comment:
I would remove "if (!isPaused())", the check is already done in printPause().
I would also put a blank line before printPause (as done in other invokations in interfaceCmd) just to highlight that printPause is an external function

Reply:
To see the source of the function I would right-click on it and select `Goto to Definition`.
Frankly speaking, I am very much confused with the rules you prefer to put line spacing between code line. How do you identify what the line spacing was for? it just ends up with spacing on every line. I prefer simple spacing rule. Line spacing between variable assignments and function calls.

Reply:
different style. In any case, if you use your style apply it to the rest of the file. So remove the blank lines where still present